lib/application/constants/strings.dart

```diff
@@ -0,0 +1,16 @@
+// This is a temporary constants file related to all strings
+//
+// TODO(matuella): update this approach to one that takes the localization (BuildContext)
+//
+// Even if the v1 for memo will only be available in ptBR, splitting random strings throughout the code will make it
+// hard when we decide to support multipla locales.
+
+const collectionsNavigationTab = 'Coleções';
+const progressNavigationTab = 'Progresso';
+
+const collectionsExploreTab = 'Explorar';
+const collectionsReviewTab = 'Revisar';
+
+const collectionsSectionHeaderSeeAll = 'VER TODOS';
```

Comment:
Wouldn't it be better to keep the responsibility of using this text in upper case for the widget that uses it?

Reply:
Agreed!